src/control/keyboard_config.cpp

```diff
@@ -56,6 +57,14 @@ KeyboardConfig::read(const ReaderMapping& keymap_lisp)
   {
     keymap_lisp.get("jump-with-up", jump_with_up_kbd);
 
+    if(!keymap_lisp.get("playernum", playernum)){
```

Comment:
Is there a reason for this to be two `if`s ? If not please use one condition and put the comments above.